pkg/providers/aws/provider_postgres.go

```diff
@@ -336,6 +336,15 @@ func (p *PostgresProvider) reconcileRDSInstance(ctx context.Context, cr *v1alpha
 			logger.Info(statusMsg)
 			return nil, croType.StatusMessage(statusMsg), nil
 		}
+		msg = fmt.Sprintf("rds instance %s is as expected", *foundInstance.DBInstanceIdentifier)
+		logger.Infof(msg)
```

Comment:
A small nit. the log message is duplicated but removal doesn't need to delay this getting merged.

Reply:
yea I can remove it in the release pr